crates/router/src/connector/paypal.rs

```diff
@@ -1553,17 +1552,17 @@ impl api::IncomingWebhook for Paypal {
             .parse_struct("PaypalWebooksEventType")
             .change_context(errors::ConnectorError::WebhookEventTypeNotFound)?;
         let outcome = match payload.event_type {
-            PaypalWebhookEventType::CustomerDisputeCreated
-            | PaypalWebhookEventType::CustomerDisputeResolved
-            | PaypalWebhookEventType::CustomerDisputedUpdated
-            | PaypalWebhookEventType::RiskDisputeCreated => Some(
+            PaypalWebhookEventType::CustomerDisputeResolved => Some(
```

Comment:
We should check the outcome code for all the Dispute events right, why is this change required here?

Reply:
They r sending outcome_code for resolved events only